static/app/components/asyncComponent.tsx

```diff
@@ -14,7 +14,7 @@ import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
 import PermissionDenied from 'sentry/views/permissionDenied';
 import RouteError from 'sentry/views/routeError';
 
-type AsyncComponentProps = Partial<RouteComponentProps<{}, {}>>;
+export type AsyncComponentProps = Partial<RouteComponentProps<{}, {}>>;
```

Comment:
Why not just keep using `AsyncComponent['props']` for now, I don't think there's a type hit here is there?